lab_6/task_6_4.py

```diff
@@ -0,0 +1,7 @@
+def replace():
```

Comment:
we need to talk with you about naming of variables.
please ask me during the class

Reply:
this variant of lab 6 is old, new variant in other branch, where a lot of troubles are fixed, include naming and io import

Reply:
I delete this branch and move lab6 in other branch, because some bug or error won't me fix git coflict. git pull doesn't work